### PR TITLE
default scheduler too busy threshold is too small

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
In benchmark case, there are too many `scheduler is busy` errors, the loads of machines are far from heavy indeed.

### CODE REVIEW

File: src/storage/config.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ const DEFAULT_GC_RATIO_THRESHOLD: f64 = 1.1;
 const DEFAULT_SCHED_CAPACITY: usize = 10240;
 const DEFAULT_SCHED_MSG_PER_TICK: usize = 1024;
 const DEFAULT_SCHED_CONCURRENCY: usize = 102400;
-const DEFAULT_SCHED_TOO_BUSY_THRESHOLD: usize = 1000;
+const DEFAULT_SCHED_TOO_BUSY_THRESHOLD: usize = 4000;
```
Comment (siddontang) [inline src/storage/config.rs]: seem we should add this to the config file too.

**Comment by siddontang** [comment]
Rest LGTM

**Comment by siddontang** [comment]
PTAL @disksing

**Comment by siddontang** [comment]
CI failed @zhangjinpeng1987

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
CI still failed @zhangjinpeng1987

**Pull request merged**